priv/repo/seeds/seeder.ex

```diff
@@ -0,0 +1,66 @@
+defmodule BikeBrigade.Repo.Seeds.Seeder do
+  alias BikeBrigade.{
+    Delivery,
+    LocalizedDateTime,
+    Riders
+  }
+
+  alias BikeBrigade.Repo.Seeds.Toronto
+
+  def program() do
+    {:ok, %Delivery.Program{}} =
+      Delivery.create_program(%{
+        name: Faker.Company.name(),
+        start_date: LocalizedDateTime.now()
+      })
+  end
+
+  def campaign() do
+    current_week =
+      LocalizedDateTime.today()
+      |> Date.beginning_of_week()
+
+    {:ok, campaign} =
+      Delivery.create_campaign(%{
+        # key :name not found in: %BikeBrigade.Delivery.Campaign{…}
+        name: Faker.Company.name(),
+        delivery_start: current_week |> start_of_day(),
+        delivery_end: current_week |> Date.add(6) |> end_of_day(),
+        location: Toronto.random_location()
+      })
+
+    campaign
+  end
+
+  def rider_for_campaign(campaign) do
+    rider =
+      Riders.list_riders()
+      |> Enum.random()
+
+    # validate_required([:campaign_id, :rider_id, :rider_capacity, :enter_building, :token])
+    Delivery.create_campaign_rider(%{
+      campaign_id: campaign.id,
+      rider_id: rider.id
+    })
```

Comment:
Should the other fields be required for seeded data as well?

Reply:
I think if the fields are not there this would fail right?

Reply:
nevermind they have defaults :)